core/src/main/java/org/apache/servicecomb/core/handler/ShutdownHookHandler.java

```diff
@@ -56,6 +57,8 @@ public long getActiveCount() {
   @Override
   public void handle(Invocation invocation, AsyncResponse asyncResp) throws Exception {
     if (shuttingDown) {
+      System.out.println("shutting down in progress");
```

Comment:
Do not use System.out.println to print logs. And we have already throw an exception with this message, and I think this log is not necessary.

Reply:
So sorry is for debug,thanks!

Reply:
Deleted